site/docs/v1/tech/guides/webauthn.adoc

```diff
@@ -0,0 +1,504 @@
+
+== Troubleshooting
+
+Due to the server- and client-side components of each of the WebAuthn ceremonies and compatibility depending on both the browser and operating system, troubleshooting issues can be difficult. These tips may help you when issues come up.
+
+=== Browser Compatibility
+
+WebAuthn is now supported across a variety of web browsers, but if you experience issues, it's worth checking whether your browser is https://caniuse.com/webauthn[compatible]. Be sure to check the notes section for limitations based on specific authenticators or operating systems and evaluate whether you may be experiencing one of those limitations.
+
+=== Secure Context
+
+WebAuthn can only be used in https://w3c.github.io/webappsec-secure-contexts/#secure-contexts[secure contexts]. Briefly, this means that the browser must have a valid TLS connection for the current page. If WebAuthn is called from an `iframe`, then both the `iframe` and the embedding page must be secured.
+
+=== Relying Party Id
+
+A WebAuthn passkey is scoped to a specific relying party Id at the time it is created. Passkeys cannot be used with a different relying party Id than they were registered, and there are restrictions on allowed relying party Ids during a WebAuthn ceremony based on the browser request origin's effective domain.
+
+If the relying party Id for a tenant is changed because of a configuration change or a domain change, passkeys registered prior to the change will no longer work. See link:/docs/v1/tech/admin-guide/webauthn#relying-party-id[Relying Party Id] in the WebAuthn Admin Guide for more detail on constraints and configuration options.
+
+=== WebAuthn JavaScript Binary Format
+
+As mentioned earlier in this guide, certain request and response fields on FusionAuth's link:/docs/v1/tech/apis/webauthn[WebAuthn APIs] must be encoded as base64url strings for transport over the network, but the WebAuthn JavaScript API uses JavaScript's `ArrayBuffer` type for these values. See the translation sections in this guide for more information on which fields must be converted.
```

Comment:
What is the translation section? I didn't see those. Do you mean the `Transform` sections?